server.py

```diff
@@ -12,11 +12,12 @@
 from flask_cors import CORS
 from flask_compress import Compress
 
+KRAKEN_VERSION = str(os.getenv('KRAKEN_VERSION'))
 SETTINGS_FILENAME = 'settings.json'
 DOA_FILENAME = 'DOA_value.html'
-DOA_PATH = '/home/krakenrf/krakensdr_doa/krakensdr_doa'
-SETTINGS_FILE = f'/{DOA_PATH}/{SETTINGS_FILENAME}'
-DOA_FILE = f'{DOA_PATH}/_android_web/{DOA_FILENAME}'
+DOA_PATH = str(os.getenv('DOA_PATH','/home/krakenrf/krakensdr_doa/krakensdr_doa'))
+SETTINGS_FILE = f'{DOA_PATH}/_share/{SETTINGS_FILENAME}'
```

Comment:
For all I see, the Kraken version with "_shared" path isn't released yet as of Aug 31st.
So we need to have a backwards compatibility option for this, since it may be downloaded and used before the next Kraken release.

Reply:
It is suitable?